workshops/semantic-kernel-ws/workshop.md

```diff
@@ -0,0 +1,393 @@
+---
+published: false                        # Optional. Set to true to publish the workshop (default: false)
```

Comment:
need to be true if you want it listed in the search page

Reply:
good point :)